db/migrate/20250909095902_make_batch_team_nullable.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class MakeBatchTeamNullable < ActiveRecord::Migration[8.0]
+  def up
+    change_column_null :batches, :team_id, true
+  end
+
+  def down
+    change_column_null :batches, :team_id, false
+  end
```

Comment:
```suggestion
  def change
    change_column_null :batches, :team_id, true
  end
```

Rails will automatically add the `down` version for you.

Reply:
Snazzy. That's very _rails_.